ci/workflows: windows release fail.

```bash
Line |
   2 |  … cho "No release-relevant changes detected. Skipping build." && exit 0
     |                                                                   ~~~~
     | The term 'exit' is not recognized as a name of a cmdlet, function, script file, or executable program. Check the
     | spelling of the name, or if a path was included, verify that the path is correct and try again.
Error: Process completed with exit code 1.
```

--- a/.github/workflows/windows_release.yml
+++ b/.github/workflows/windows_release.yml
@@ -42,7 +42,7 @@ jobs:
 
     - name: Skip build if only docs or irrelevant files changed
       if: steps.changes.outputs.release != 'true'
-      run: echo "No release-relevant changes detected. Skipping build." && exit 0
+      run: echo "No release-relevant changes detected. Skipping build." && $(exit 0)
 
     - name: Install Rust toolchain
       uses: dtolnay/rust-toolchain@stable

